### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by yvette809**
reviews

- I have created  reviews for the specific movie
-  first page shows 5 reviews as instructed
- Created buttons to show the next and previous reviews

### CODE REVIEW

**Comment by yvette809** [comment]
screen2

**Review by holst9559 — APPROVED** [review]
Looks good, great job!

Nothing to comment on really, I now notice that I missed to format alot of code in my project. I need to make my prettier format on save, thought I had set it to do so but apparently not....

File: static/js/getReviews.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+// pagination function
+function paginate(reviews) {
+  console.log("paginatereviews", reviews);
+
+  const numOfReviewsPerPage = 5;
+  const numberOfPages = Math.ceil(reviews.length / numOfReviewsPerPage);
+  const newReviews = Array.from({ length: numberOfPages }, (_, index) => {
+    const start = index * numOfReviewsPerPage;
+    return reviews.slice(start, start + numOfReviewsPerPage);
+  });
```
Comment (solidtop) [inline static/js/getReviews.js:77]: A thought on the pagination. If I understand this correctly the pagination is handled on the client? 
An alternative to this is to handle how many/what reviews to show on moviePage on the backend/server. The logic can be run on our server and then only send in this case 5 reviews to client. And maybe use querystrings in the path like ?page=2 to cycle through the pages. What do you think? :)

Comment (yvette809) [reply static/js/getReviews.js:77]: I think it is a matter of preference. Even if you do the logic on the server, you will still have to do some work in the client side to make it work. If you send just 5 reviews to client as you said, then you won't get more reviews to show on the subsequent pages as stated in the task

Comment (solidtop) [reply static/js/getReviews.js:77]: I agree! Although with querystring you can change what page to show from the client. For example everytime you click on nextPage or previousPage button/link it sends a new request to our server  with querystring ?page=pageIndex where pageIndex is the page you want to display. 
This is just an alternative and the current implementation works good! Good job :)

**Review by solidtop — APPROVED** [review]
Looks good! Simple and nice styling that shows everything a review needs.
Also nice that you formatted the code, will make sure to keep the indents the same 👍

File: static/js/getReviews.js (hunk 1 above)
Comment (yvette809) [reply static/js/getReviews.js:77]: That's true. That is an alternative solution. I will think about that next time

Comment (yvette809) [reply static/js/getReviews.js:77]: I asked Richard about what you suggested and he thinks that's the most appropriate solution. I will try to implement that and see how it goes

**Pull request merged**